src/patternfly/components/Nav/nav.scss

```diff
@@ -360,6 +365,20 @@
       background-color: var(--pf-c-nav__simple-list-link--m-current--BackgroundColor);
     }
 
+    &.pf-m-separator {
+      position: relative;
```

Comment:
do you think it's worth adding `margin-bottom: 1px` here, and setting `bottom` of the pseudo element to `-1px` so that the separator is between the links and not in the link the class is applied to?

Reply:
oh ya I like it! Have to change it to `-1 *...`